libraries/RC_Channel/RC_Channel.h

```diff
@@ -181,6 +181,7 @@ class RC_Channel {
         DISARM =              81, // disarm vehicle
         Q_ASSIST =            82, // disable, enable and force Q assist
         ZIGZAG_Auto =         83, // zigzag auto switch
+        FWD_THR =             84, // VTOL manual forward throttle
```

Comment:
I have been adding manual control analog inputs at the end to line up with where the four main ones will end up when we finally kill RCMAP. So this would be 209

Reply:
thanks, changed to 209